extensions/amp-gist/amp-gist.md

```diff
@@ -47,13 +47,24 @@ Example:
 </amp-gist>
 ```
 
+### Single file
+
+```html
+<amp-gist
+    data-gistid="a19e811dcd7df10c4da0931641538497"
+    data-file="hi.c"
+    layout="responsive"
+    width="480" height="270">
+</amp-gist>
+```
+
 ## Behavior
 
 This extension creates an iframe and displays the gist from GitHub.
 
 ## Attributes
 
-It requires the `data-gistid` attribute of the gist.
+It requires the `data-gistid` attribute of the gist. The `data-file` is used for displaying only one file in a gist and it is optional.
```

Comment:
let's format this differently:

These are the valid attributes for the `amp-gist` component:

**data-gistid** (required)
The Id of the gist to embed.

**data-file** (optional)
`data-file` is used for displaying only one file in a gist.